Plugins/Json/src/AmbiguityConfigJsonConverter.cpp

```diff
@@ -35,15 +35,26 @@ void from_json(const nlohmann::json& j, ConfigPair& p) {
 
     detectorConfig.sharedHitsFlag = value["sharedHitsFlag"];
 
-    std::vector<double> factorHits = value["factorHits"];
-    std::vector<double> factorHoles = value["factorHoles"];
+    std::vector<double> goodHits = value["goodHits"];
+    std::vector<double> goodHoles = value["goodHoles"];
```

Comment:
are we ok with copies?

Reply:
what about having these as `const` as well?

```suggestion
    const std::vector<double>& goodHits = value["goodHits"];
    const std::vector<double>& goodHoles = value["goodHoles"];
```